sound/soc/codecs/rt711-sdca-sdw.c

```diff
@@ -225,6 +225,14 @@ static int rt711_sdca_read_prop(struct sdw_slave *slave)
 		j++;
 	}
 
+	prop->dp0_prop = devm_kzalloc(&slave->dev, sizeof(*prop->dp0_prop),
+				      GFP_KERNEL);
+	if (!prop->dp0_prop)
+		return -ENOMEM;
+
+	prop->dp0_prop->simple_ch_prep_sm = true;
+	prop->dp0_prop->ch_prep_timeout = 10;
```

Comment:
No other codec can declare this? Only rt711-sdca?

Reply:
We get the properties from sdw_slave_read_prop() for other codecs.

Reply:
I mean in theory rt-715, 721, 722 could also?
What I meant is: why only rt711?

Reply:
In theory, most SDW codecs support BPT. In fact, prop->dp0_prop->simple_ch_prep_sm should come from the "mipi-sdw-simplified-channel-prepare-sm" property. Only rt711 is because it is the first codec that we tested. I will get the supported codec list from Realtek and Cirrus and add the codecs to the PR.